docs/getting-started/_quickstart-parts/_quickstart_dotnet.mdx

```diff
@@ -32,6 +32,30 @@ Permit permit = new Permit(
     "http://localhost:7766"
 );
 
+// you can also set more config options
+Permit permitClient = new Permit("[YOUR_API_KEY]",
+                                    // the URL of the Permit.io PDP container
+                                    pdp: "http://localhost:7766",
+                                    // the tenant to use if the tenant is not provided
+                                    defaultTenant: "default",
+                                    // whether to use the default tenant if the tenant is not provided
+                                    useDefaultTenantIfEmpty: true, 
+                                    // should run in debug mode
+                                    debugMode: true,
+                                    // set the log level
+                                    level: "info",
+                                    // set the log label
+                                    label: "Permitio-sdk",
+                                    // should log as JSON
+                                    logAsJson: false,
+                                    // the URL of the API (relevant for EU customers)
+                                    apiURL: "https://api.eu-central-1.permit.io",
+                                    // should raise errors (instead of the default behavior of returning false for network errors in permit checks)
+                                    raiseErrors: false,
+                                    //the environment_id and project_id to avoid getting them from the cloud API
```

Comment:
```suggestion
                                    // Optional: manual set the environment_id and project_id for the SDK client (to avoid `scope` api call)
```